.github/workflows/linter.yml

```diff
@@ -11,14 +11,11 @@ name: Lint Code Base
 # https://help.github.com/en/articles/workflow-syntax-for-github-actions
 #
 
-#############################
-# Start the job on all push #
-#############################
+###################################
+# Start the job on a pull request #
+###################################
 on:
-  pull_request:
-    branches:
-      - master
-      - 2.*-develop
+  pull_request
```

Comment:
So we run the check on all branches, but we only require a passing check on protected branches, right? We configure that with the repo settings instead of the GitHub Action workflow config.

Reply:
Correct.